aarch64/src/devcons.rs

```diff
@@ -60,8 +60,8 @@ const MBOX_EMPTY: u32 = 0x4000_0000;
 
 // Delay for count cycles
 fn delay(count: u32) {
-    #[cfg(not(test))]
     for _ in 0..count {
+        #[cfg(not(test))]
```

Comment:
Actually, I don't see any reason not to remove this entirely.

Reply:
yeah, funnily I was wondering the same thing right now